modules/social_features/social_event/modules/social_event_invite/src/Form/EnrollInviteEmailForm.php

```diff
@@ -116,22 +115,22 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     ];
 
     // Load event invite configuration.
-    $invite_config = $this->configFactory->get('social_event_invite.settings');
+    $invite_config = $this->configFactory()->get('social_event_invite.settings');
```

Comment:
@navneet0693 I would recommend to define a variable for $this->configFactory() and use the variable instead of the class property; especially when it's being used multiple times in the same method.

Reply:
Okay.